release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1694,6 +1694,25 @@ In the following tables, features are marked with the following statuses:
 
 * When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
 
+* When you run CNF latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
```

Comment:
@linuxqiao Can you provide lgtm on this OCP 4.15 known issue description for https://issues.redhat.com/browse/RHEL-9279 please
This is a direct copy of the 4.14 Knonw Issues note

Reply:
/lgtm
These issues have not yet been resolved